landing-page/content/about/about.html

```diff
@@ -23,8 +23,8 @@
 <div class="button-box">
 <ul class="list-inline intro-social-buttons">
     <li>
-        <a href="/getting-started" class="btn btn-default btn-lg">
-            <span class="network-name">Learn More</span>
+        <a href="/quickstarts" class="btn btn-default btn-lg">
```

Comment:
Should this be `spark-quickstart` since we're going directly to that one?

Reply:
Fixed! Here's the menu entry added to the topnav:
```
{ name = "Quickstart", url = "/spark-quickstart", weight = 100 },
```